mesheryctl/internal/cli/root/system/logs.go

```diff
@@ -41,8 +41,10 @@ import (
 var logsCmd = &cobra.Command{
 	Use:   "logs",
 	Short: "Print logs",
-	Long:  `Print history of Meshery's logs and begin tailing them.`,
-	Args:  cobra.ArbitraryArgs,
+	Long: `Print history of Meshery's logs and begin tailing them for the Kubernetes platform.
+
+It also shows the tailing logs of a specific component e.g. "mesheryctl system logs meshery-istio"`,
```

Comment:
This could be moved to an examples section right? And other general use cases could also be added.

In this example, this would not tail the logs as this is for the Kubernetes platform. We have to add this tailing feature as we discussed.